_posts/2022-09-10-executive-director-pyopensci-leah-wasser-updates.md

```diff
@@ -0,0 +1,123 @@
+---
+layout: single
+title: "Big Changes for pyOpenSci - an update from the new executive director, leah wasser, what's next?"
+excerpt: "pyOpenSci has recently transitioned to a fiscal sponsor and has a new executive director - Leah Wasser. Learn more about our goals over the next few years and how to get involved with the community."
+author: "Leah Wasser"
+permalink: /blog/new-executive-director-leah-wasser
+header:
+    overlay_color: "#666"
+    overlay_filter: 0.6
+categories:
+  - updates
+  - blog-post
+---
+
+
+## pyOpenSci is now an independent community organization
+
+I am thrilled to announce that pyOpenSci is now a fiscally sponsored project of 
+Community Initiatives! This means that pyOpenSci now serves the community of 
```

Comment:
Make "Community Initiatives" a link for people that are not familiar with the org?